src/main/java/com/launchdarkly/client/FeatureFlag.java

```diff
@@ -175,12 +174,10 @@ VariationOrRollout getFallthrough() {
   static class EvalResult {
     private JsonElement value;
     private List<FeatureRequestEvent> prerequisiteEvents;
-    private Set<String> visitedFeatureKeys;
 
-    private EvalResult(JsonElement value, List<FeatureRequestEvent> prerequisiteEvents, Set<String> visitedFeatureKeys) {
+    private EvalResult(JsonElement value, List<FeatureRequestEvent> prerequisiteEvents) {
```

Comment:
I don't even think this class is necessary anymore. Can't you just collect the events via the list passed as the function argument?